image-index.md

```diff
@@ -67,7 +67,12 @@ For the media type(s) that this document is compatible with, see the [matrix][ma
 
     - **`features`** *array of strings*
 
-        This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature (for example `sse4` or `aes`).
+        This OPTIONAL property specifies an array of strings, each specifying a mandatory CPU feature.
+
+        When `architecture` is 386 or amd64 Image indexes SHOULD use, and implementations SHOULD understand, values [supported by Linux][cpufeatures.h] with the `X86_FEATURE_` prefix removed and the remainder lowercased (e.g. `fpu` for `X86_FEATURE_FPU` and `vme` for `X86_FEATURE_VME`).
+        On Linux on these architectures, the features supported by host CPUs can be found in the `flags` entries in the `cpuinfo` file provided by the [proc filesystem][proc.5].
+
+        When `architecture` is neither 386 nor amd64, values are implementation-defined.
```

Comment:
@AkihiroSuda, in dc5e4a0 → e1045bf, I've picked up your backticks, “Image” → “image”, and commas.  I've also picked up your “SHOULD be submitted”, but kept my “implementation-defined” for the reasons I gave [here][1].

[1]: https://github.com/opencontainers/image-spec/pull/631#issuecomment-291220772